src/main/java/com/odysseusinc/arachne/portal/config/WebSecurityConfig.java

```diff
@@ -73,7 +73,7 @@
 
 @Configuration
 @EnableWebSecurity
-@EnableGlobalMethodSecurity(prePostEnabled = true)
+@EnableGlobalMethodSecurity(prePostEnabled = true, securedEnabled = true)
```

Comment:
How Spring combines this with the same annotation defined in `AccessConfig`?

Reply:
there are 2 ways:
1) to remove `@EnableGlobalMethodSecurity` from WebSecurityConfig.java (I don't understand why it was added here), and then `securedEnabled` from AccessConfig.java will apply
2) to add `securedEnabled = true` in WebSecurityConfig.java

Reply:
What I am saying is that the second option, which you implemented, looks a bit confusing and I personally don't get how the multiple `EnableGlobalMethodSecurity` are combined. Therefore, I would propose going for the first option (just re-test that nothing breaks)

Reply:
@pavgra ok, will implement the first, I already tested both cases and it behaves properly